packages/contracts/src/Admin.sol

```diff
@@ -30,9 +31,11 @@ contract Admin is IMembership, PluginCloneable, ProposalUpgradeable {
     /// @notice Initializes the contract.
     /// @param _dao The associated DAO.
     /// @dev This method is required to support [ERC-1167](https://eips.ethereum.org/EIPS/eip-1167).
-    function initialize(IDAO _dao) external initializer {
+    function initialize(IDAO _dao, TargetConfig calldata _targetConfig) external initializer {
         __PluginCloneable_init(_dao);
 
+        _setTargetConfig(_targetConfig);
+
```

Comment:
Shouldn't we use `reintializer(n)` here, and then create another function for initializing from other versions?

Reply:
Admin is non-upgradeable but cloneable, so no..

Reply:
true